### ISSUE TITLE
OVRManager doesn't take position of the camera

### ISSUE THREAD

**Issue by ndgreen** [TonyViT/MrcXrtHelpers#2]
Despite any positioning/rotation in the scene editor when run (about a half second into the run) the camera reverts to a default position. Not sure how to reset the OVRCam back to the viewpoint it is set to in the editor

**Comment by ndgreen** [comment]
For context it is in RemoveMRCamerasTracking right after
 var waiter = new WaitForSeconds(0.5f);

If that is increase so is the time before it locks to a position. So for whatever reason with the TrackedPose is destroyed it gets set to this default position that I'm not sure where it comes from (coordinate wise).

**Comment by ndgreen** [comment]
After debugging some more: If I just remove all this and go base OVRCamRig without XR toolkit the same thing happens so I think when the cams are removed it goes back to this default static position. I'm just not sure how OVRManager is determining this static position.

**Comment by TonyViT** [comment]
Hello @ndgreen , what version of Unity are you using? Can this feedback I have been sent help you?

_I’m using Unity 2020.3 and Action Based XR Interaction Toolkit. At first, I found the problem where the camera keeps following my headset rotation when I turned on the OBS Studio. After debugging for several hours, I found the problem and fixed it.  You just need to change the import library from using UnityEngine.SpatialTracking to UnityEngine.InputSystem.XR on RemoveMRCamerasTracking.cs I hope it can help other people too who encounter the same problem_
